### DISCUSSION TITLE
$newModel::create(['tags' => ['first tag']) not work

### DISCUSSION THREAD

**Discussion by nekooee** [spatie/laravel-tags#316] [Q&A]
Hi,
my code is:
```
class Article extends Model
{
    use Translatable;
    use Sluggable;
    use HasTags;
    .....
----------------------------------
class ArticleController extends Controller
{
 public function store(StoreArticleRequest $request)
    {
 $newsItem = Article::create([
            'title' => 'The Article Title',
            'tags' => ['first tag', 'second tag'],
        ]);
        ....
```
But no tags are created in the database.
please help me.
I'm in a hurry and I'm stuck.

**Comment by leolll** [comment]
Here's how I resolved this.  Using the example model of **NewsItem** from the docs, make sure to add **tags** to the fillable property:

```
class NewsItem extends Model
{
    use HasFactory, HasTags;

    public $fillable = ['name', 'tags'];
}
```

**Reply by nekooee** [reply]
I did not enter in fillable. Because this is a relation and not a column in the table. But if you tested it and it was correct then I should do the same.
thank you.
